packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -373,10 +376,13 @@ const DownloadOptionsBox = ({
   iconUrl: string | (() => Promise<string>);
   iconBackground?: string;
 }) => {
+  const isBrowserCard = !accentColor;
```

Comment:
Since the `accentColor` property is optional on the `Wallet` type, this means some wallets could inadvertently set `isBrowserCard` to true? Would it maybe be safer to have an explicit prop for this rather than inferring it? e.g. `type="browser"` / `type="app"`?